src/EditorFeatures/Core/Diagnostics/AbstractDiagnosticsTaggerProvider.cs

```diff
@@ -153,9 +154,6 @@ protected internal virtual ImmutableArray<DiagnosticDataLocation> GetLocationsTo
                 // occasions
                 return;
             }
-
-            static SnapshotSpan GetDiagnosticSnapshotSpan(DiagnosticDataLocation diagnosticDataLocation, ITextSnapshot editorSnapshot, SourceText sourceText)
-                => DiagnosticData.GetExistingOrCalculatedTextSpan(diagnosticDataLocation, sourceText).ToSnapshotSpan(editorSnapshot);
```

Comment:
note: this used to use the .SourceText property on location if present.  Now it always computes given the SourceText provided.  THis means we always do need a SourceText to make this determination.  HOwever, checking all product code that called into this, we should always have a SourceText (or be able to get one cheaply), so this seems like not a problem.